masterlist.yaml

```
@@ -887,3 +887,27 @@ plugins:
       - 'Imperial Legion Goods.esp'
       - 'Legionpapers.esp'
       - 'Publius_Claudius_Follower.ESP'
+
+  - name: 'Thieves_Guild_Overhaul.ESP'
+    url:
+      - link: 'https://www.nexusmods.com/morrowind/mods/49944/'
+        name: 'Thieves'' Guild Overhaul'
+    inc:
+      - 'Bal Molagmer Add-on.ESP'
+      - 'Lore-Friendly Bal Molagmer Gloves.ESP'
+      - 'More Redoran Master Helms.ESP'
+      - 'Plunder the Dungeon.ESP'
+      - 'Smooth Moves - Romancing Ahnassi.esp'
+      - 'Thieves'' Guild in Gnaar Mok.esp'
+      - 'Wanted Posters.ESP'
+  - name: ' Thieves_Guild_Overhaul - Purist Ahnassi Romance version.ESP'
+    url:
+      - link: 'https://www.nexusmods.com/morrowind/mods/49944/'
+        name: 'Thieves'' Guild Overhaul'
+    inc:
+      - 'Bal Molagmer Add-on.ESP'
+      - 'Lore-Friendly Bal Molagmer Gloves.ESP'
+      - 'More Redoran Master Helms.ESP'
+      - 'Plunder the Dungeon.ESP'
+      - 'Thieves'' Guild in Gnaar Mok.esp'
+      - 'Wanted Posters.ESP'
```

I corrected a few inconsistencies with plugin names in your PR & as they're downloaded. I've also refactored it with a regex plugin. Also, since the alternate version contains a stripped down version of _Smooth Moves_, you can say it's incompatible with it as well. Finally, I added a `useOnlyOneX` message as only one of the plugins should be used at a time:

```suggestion
  - name: 'Thieves_Guild_Overhaul( - Purist Ahnassi Romance version)?\.ESP'
    url:
      - link: 'https://www.nexusmods.com/morrowind/mods/49944/'
        name: 'Thieves'' Guild Overhaul'
    inc:
      - 'Bal Molagmer Add-on.ESP'
      - 'Lore-friendly Bal Molagmer Gloves.ESP'
      - 'More Redoran Master Helms.ESP'
      - 'Plunder the Dungeon.ESP'
      - 'Smooth Moves - Romancing Ahnassi.esp'
      - 'Thieves Guild in Gnaar Mok.ESP'
      - 'Wanted Posters.ESP'
    msg:
      - <<: *useOnlyOneX
        subs: [ 'Thieves'' Guild Overhaul ESP' ]
        condition: 'many("Thieves_Guild_Overhaul( - Purist Ahnassi Romance version)?\.ESP")'
```